docs/dashboards/types-of-components.md

```diff
@@ -0,0 +1,42 @@
+Types of components
+===
+
+Components are the building blocks of the dashboard layout. There are numbers of types of components, which you can use out of the box. Some of them have come with some default configuration, and some components which leave you some flexibility so that you can configure them all by yourself.  You define which type of the component you want to use by defining its “type” property in the configuration object.
```

Comment:
`Some of them have come with some default configuration, and some components which leave you some flexibility so that you can configure them all by yourself.`

Perhaps, we should define which components are which.